cli/flags.go

```diff
@@ -295,6 +295,10 @@ var ioFlags = []cli.Flag{
 		Usage: "Add checksum to uploaded object. Values: CRC64NVME, CRC32[-FO], CRC32C[-FO], SHA1 or SHA256. Requires server trailing headers (AWS, MinIO)",
 		Value: "",
 	},
+	cli.BoolFlag{
+		Name:  "single-delete",
+		Usage: "Delete objects one at a time instead of using bulk delete",
```

Comment:
```suggestion
		Usage: "Delete objects one at a time instead of using bulk delete",
		Hidden: true,
```
I don't mind it, but it feels like a niche workaround for incomplete S3 support.